### PR TITLE
Update configuration to have "cosbeta" point at actual cos-beta image again

### PR DESCRIPTION
**Pull request by filbranden**
@yguo0905

### CODE REVIEW

File: experiment/test_config.yaml [hunk 1]
```diff
@@ -1027,7 +1026,7 @@ nodeCommon:
 nodeImages:
   cosbeta:
     args:
-    - --node-args=--images=cos-stable-63-10032-71-0-p
+    - --node-args=--images=cos-beta-64-10176-60-0
```
Comment (yguo0905) [inline experiment/test_config.yaml]: FYI, I'm working on https://github.com/kubernetes/test-infra/issues/3320 to avoid updating the image everytime.

Comment (filbranden) [reply experiment/test_config.yaml]: Nice!

But for now... This looks correct, right?

Or should we skip this update for now?

Thanks!
Filipe

Comment (yguo0905) [reply experiment/test_config.yaml]: Yeah, this is good. Already LGTM'ed.

**Comment by filbranden** [comment]
@krzyzacy Can you please take a look? Thanks!

**Comment by krzyzacy** [comment]
/approve
please rebase

**Comment by filbranden** [comment]
> please rebase

Done. PTAL

**Comment by krzyzacy** [comment]
it will be automatically merged once all presubmit passes

**Pull request merged**